README.md

```diff
@@ -196,54 +174,37 @@ Due to its nature, a stream's stack trace might have dozens of frames, which, mo
 A naive way of debugging, is by injecting side effects into the stream, like so:
 
 ```swift
-let searchString = textField.rac_textSignal()
-    .toSignalProducer()
-    .map { text in text as! String }
-    .throttle(0.5, onScheduler: QueueScheduler.mainQueueScheduler)
+let searchString = textField.reactive.continuousTextValues
+    .throttle(0.5, on: QueueScheduler.main)
     .on(event: { print ($0) }) // the side effect
 ```
 
-This will print the stream's [events][Events], while preserving the original stream behaviour. Both [`SignalProducer`][Signal producers]
-and [`Signal`][Signals] provide the `logEvents` operator, that will do this automatically for you:
+This will print the stream's [events][Events], while preserving the original stream behaviour. Both [`SignalProducer`][SignalProducer]
+and [`Signal`][Signal] provide the `logEvents` operator, that will do this automatically for you:
 
 ```swift
-let searchString = textField.rac_textSignal()
-    .toSignalProducer()
-    .map { text in text as! String }
-    .throttle(0.5, onScheduler: QueueScheduler.mainQueueScheduler)
+let searchString = textField.reactive.continuousTextValues
+    .throttle(0.5, on: QueueScheduler.main)
     .logEvents()
 ```
 
 For more information and advance usage, check the [Debugging Techniques](Documentation/DebuggingTechniques.md) document.
 
 ## How does ReactiveSwift relate to Rx?
 
-ReactiveCocoa was originally inspired, and therefore heavily influenced, by
-Microsoft’s [Reactive
-Extensions](https://msdn.microsoft.com/en-us/data/gg577609.aspx) (Rx) library. There are many ports of Rx, including [RxSwift](https://github.com/ReactiveX/RxSwift), but ReactiveCocoa is _intentionally_ not a direct port.
-
-**Where ReactiveSwift differs from Rx**, it is usually to:
+While ReactiveCocoa was inspired and heavily influenced by [ReactiveX][] (Rx), ReactiveSwift is
+an opinionated implementation of the grand concept, and _intentionally_ not a direct port like
```

Comment:
_the grand concept_ reads a little weird. I'd take out the word _grand_ or write something like _is an opinionated implementation of reactive programming_.